assets/src/scripts/monitoring-location/components/hydrograph/selectors/time-series-data.js

```diff
@@ -150,3 +150,34 @@ export const getPreferredIVMethodID = createSelector(
         return methodMetaData[methodMetaData.length - 1].methodID;
     }
 );
+
+
+/**
+ * Returns a array of methods sorted from most to least points in the selected time span.
+ * @return {Array} Array of objects with details about the sampling methods available for the primary parameter.
+ */
+export const getSortedIVMethods = createSelector(
+    getIVData('primary'),
+    (ivData) => {
+        if (!ivData || !Object.keys(ivData.values).length) {
+            return null;
+        }
+        const methodsForPrimarySelection = Object.values(ivData.values)
+            .map(methodValues => {
+                return {
+                    pointCount: methodValues.points.length,
+                    lastPoint: methodValues.points.length ? methodValues.points[methodValues.points.length - 1] : null,
+                    methodID: methodValues.method.methodID,
+                    methodDescription: methodValues.method.methodDescription
```

Comment:
Not all methods have a description. We need to put some sort of string here. We were using "None"

Reply:
There is code that catches that case further downstream, but you are right, and it would make more sense for that to happen here.